cumulusci/tasks/bulkdata/extract.py

```diff
@@ -286,6 +290,13 @@ def _create_table(self, mapping):
         mapper_kwargs = {}
         self.models[mapping["table"]] = type(model_name, (object,), {})
 
+        # Track IsPersonAccount for Account and Contact tables if person accounts is enabled.
+        # IsPersonAccount field should never exist in a mapping because the field is not creatable.
```

Comment:
With FLS-checking changes recently merged, the operation will fail if the user tries to include this field. (Just an FYI, not a change request).

Reply:
That's good the operation will fail.  `IsPersonAccount` should never exist in the user's mapping.   cci then adds `IsPersonAccount` behind the scenes.